extensions/wikia/AssetsManager/config.php

```diff
@@ -2090,6 +2091,15 @@
 	)
 );
 
+/** DesignSystem extension */
+$config['design_system_js'] = array(
+	'type' => AssetsManager::TYPE_JS,
+	'skin' => [ 'oasis' ],
+	'assets' => array(
```

Comment:
This uses short array syntax on the skin so should be consistent, and per our conventions all the arrays should use short array syntax now anyway.